opensrp-giz-malawi/src/test/java/org/smartregister/giz/dao/ReportDaoTest.java

```diff
@@ -0,0 +1,161 @@
+        Map<String, String> locations = ReportDao.extractRecordedLocations(anmLocation1);
+
+        Mockito.verify(database).rawQuery(Mockito.anyString(), Mockito.any());
+        String expectedLocationId = null;
+        for (Map.Entry<String, String> entry : locations.entrySet()) {
+            expectedLocationId = entry.getKey();
+        }
+        Assert.assertNull(expectedLocationId);
```

Comment:
Add an assertion that `locations` is empty

Reply:
Done